### PR TITLE
[SPIRV] Fix translation of get_image_* built-in functions for image t…

### PR DESCRIPTION
**Pull request by bader**
…ypes with access qualifiers.

### CODE REVIEW

File: test/SPIRV/transcoding/image_get_size_with_access_qualifiers.ll [hunk 1]
```diff
@@ -0,0 +1,56 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.txt
+; RUN: FileCheck < %t.txt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: TypeInt [[typeint:[0-9]+]]
+; CHECK-SPIRV: TypeVector [[typevector:[0-9]+]] [[typeint]] {{[0-9]+}}
+; CHECK-SPIRV: ImageQuerySizeLod [[typevector]]
```
Comment (yxsamliu) [inline test/SPIRV/transcoding/image_get_size_with_access_qualifiers.ll]: Can we add a check for SPIR-V image type.

Comment (bader) [inline test/SPIRV/transcoding/image_get_size_with_access_qualifiers.ll]: Updated the test: remove unnecessary code and add a check for image type.

**Comment by bader** [comment]
Sam, does it look good to you?

**Comment by yxsamliu** [comment]
LGTM. Thanks!

**Pull request merged**